endpoints/openrtb2/sample-requests/invalid-whole/unknown-bidder.json

```diff
@@ -0,0 +1,38 @@
+{
+    "description": "Copy of the prebid test ad, with the addition of an unknown bidder",
+  
+    "message": "Invalid request: request.imp[0].ext contains unknown bidder: unknownbidder. Did you forget an alias in request.ext.prebid.aliases?\n",
```

Comment:
This test looks like it overlaps with `imp-ext-unknown-bidder.json`... did you mean to hit another edge case here?

Reply:
This test fails when not using the correct default alias. This test passes when you do use the correct alias. It overlaps, because I needed one valid bidder to exist so this test will still pass when #713 is fixed.

I suppose I could have just modified `imp-ext-unknown-bidder.json`, but this way we will explicitly test the case where both valid and invalid bidders are provided. It is a little counterintuitive that we use a bad request to test that in a certain case the request is valid. :)